web/src/utils/factory-management/exports.spec.ts

```diff
@@ -26,6 +28,17 @@ describe('exports', () => {
         amount: 100,
         recipe: 'IronPlate',
       })
+
+      addProductToFactory(oilFac, {
+        id: 'LiquidOil',
+        amount: 30,
+      })
+
+      addProductToFactory(oilFac, {
+        id: 'HeavyOilResidue',
+        amount: 40,
+        recipe: 'Alternate_HeavyOilResidue',
+      })
```

Comment:
This isn't a byproduct though, you're adding an actual product here.

Reply:
Need to remove this and then asset that the `factory.byProduct` contains the byproduct, AND it's still showing up for export.